src/main/java/com/savvato/tribeapp/services/CosignServiceImpl.java

```diff
@@ -100,4 +97,20 @@ public List<CosignsForUserDTO> getAllCosignsForUser(Long userIdReceiving) {
 
         return cosignsForUserDTOs;
     }
+
+    @Override
+    public Optional<GenericResponseDTO> validateCosigners(Long userIdIssuing, Long userIdReceiving) {
```

Comment:
Instead of an Optional<GenericResponseDTO>, could you just return a string containing the error message, or null if there is no error? Did we do the Optional somewhere else and you were trying to maintain consistency?

Reply:
I was using Optional to avoid null pointer errors or handling null values, but if that is preferred, I can refactor.

Reply:
Gotcha.. let's keep the Optional.